### PR TITLE
Directory structure

### PR DESCRIPTION
**Pull request by dorukozturk**

### CODE REVIEW

File: girder_worker_tasks/app_support/__init__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class ArborNovaTasksGirderWorkerPlugin(GirderWorkerPluginABC):
+    def __init__(self, app, *args, **kwargs):
+        self.app = app
+
+    def task_imports(self):
+        # Return a list of python importable paths to the
+        # plugin's path directory
+        return [
+            'app_support'
```
Comment (mgrauer) [inline girder_worker_tasks/app_support/__init__.py]: 'app_support',
'app_support2'

File: girder_worker_tasks/app_support/__init__.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+from girder_worker import GirderWorkerPluginABC
+from girder_worker.app import app
+from girder_worker.utils import girder_job
+from .pgls import pgls
```
Comment (mgrauer) [inline girder_worker_tasks/app_support/__init__.py]: from .app_support import *
from .app_suppor2 import *

Comment (mgrauer) [reply girder_worker_tasks/app_support/__init__.py]: to support import *, within app_support/__init__.py we need

__all__ = (
pgsr,
asr
)

**Comment by mgrauer** [comment]
Add back in `arbor_tasks` folder, so new folder structure is

    girder_worker_tasks/
        arbor_tasks/
             app_support
             base

And then tasks are individual python files within e.g. `app_support`

And we are ok with importing individual tasks at the level of `girder_worker_tasks/arbor_tasks/__init__.py`, e.g.

`from import .app_support.pgls import pgls`

And we have to add additional siblings (directories such as `base`) of `app_support` to `ArborNovaTasksGirderWorkerPlugin.task_imports`.

And base will have python files under, no more folders.

And setup.py will revert back.

**Comment by mgrauer** [comment]
Also update  girder_plugin/arbor_nova/rest.py  to reflect the new directory structure for importing girder_worker tasks.

**Comment by dorukozturk** [comment]
@curtislisle @mgrauer I just made the changes that we talked about. 

So in order to create a new task under let's say core 2 things need to be done:
1) Create a new python file in arbor_nova_tasks/arbor_tasks/core/
2) Import the task function here arbor_nova_tasks/arbor_tasks/core/__init__.py

If you want to create a new task type (lets call it foo) and a new task (bar) you need to do 5 things:
1) Create a new directory arbor_nova_tasks/arbor_tasks/foo
2) Create arbor_nova_tasks/arbor_tasks/foo/__init__.py
3) Create your task file /arbor_nova_tasks/arbor_tasks/foo/bar.py
4) Import the task function from bar.py inside the arbor_nova_tasks/arbor_tasks/foo/__init__.py
5) Add an entry to import_tasks list in arbor_nova_tasks/__init__.py  'arbor_nova_tasks.arbor_tasks.foo'

**Comment by curtislisle** [comment]
Thanks for this PR. The description of the process makes sense.  Even to me 😎

**Pull request merged**